README.md

```diff
@@ -38,3 +38,37 @@ Demo
 
 With the server running locally, 
 now navigate to: https://martindurant.pyscriptapps.com/empty-tundra/latest/
+
+Installation with Optional Dependencies (fsspec-proxy)
+-----------------------------------------------------
+
+The following steps apply only to the `fsspec-proxy` package. The package has
+several optional dependency groups:
+
+- `s3`: Required for S3 access (needed for the "Conda Stats" example)
+- `anaconda`: Required for Anaconda Cloud access
+- `all`: All optional dependencies
+
+To install dependencies from Anaconda Cloud (like `anaconda-cloud-storage`),
+configure pip to use the Anaconda Cloud index as an extra source. Create (or
+edit) the file `~/.config/pip/pip.conf` (on macOS/Linux) or
+`%APPDATA%\pip\pip.ini` (on Windows) and add:
+
+    [global]
+    extra-index-url = https://pypi.anaconda.org/anaconda-cloud/simple
```

Comment:
It's possible to add this as a command line arg, right?

Reply:
Hi @martindurant, thanks for the feedback, I added a 2nd method with the CLI argument.